okhttp/src/main/java/com/squareup/okhttp/ConnectionSpec.java

```diff
@@ -219,7 +219,7 @@ private ConnectionSpec supportedSpec(SSLSocket sslSocket) {
     private String[] tlsVersions;
     private boolean supportsTlsExtensions;
 
-    Builder(boolean tls) {
+    public Builder(boolean tls) {
```

Comment:
This is an unfortunate constructor to expose publicly. At least with methods that take a boolean you're afforded a bit of context as to what the value corresponds to. I mean, would anyone even want to call this with `false` publicly?